### ISSUE TITLE
Wallet initialization popup closes without user's input

### ISSUE THREAD

**Issue by JKKGBE** [ergoplatform/ergo#1258]
Ergo
```
v4.0.0-alpha
```
Browser 
```
Chrome version 87.0.4280.88 (Official Build) (64-bit)
```
OS
```
Operating System: Arch Linux
Kernel: Linux 5.4.83-1-lts
Architecture: x86-64
```

In my case it closed on two occasions after initializing with an empty password, when presented the mnemonic.

**Comment by deadit** [comment]
Thank you, will take a look!

**Comment by kushti** [comment]
fixed in 4.0.0 release
